chat-plugins/thing-of-the-day.js

```diff
@@ -5,9 +5,9 @@ const FS = require('./../lib/fs');
 const MINUTE = 60 * 1000;
 const YEAR = 365 * 24 * 60 * MINUTE;
 
-const theStudio = Rooms.get('thestudio');
-const tvbf = Rooms.get('tvbooksfilms');
-const yt = Rooms.get('youtube');
+const theStudio = /** @type {ChatRoom} */ (Rooms.get('thestudio'));
+const tvbf = /** @type {ChatRoom} */ (Rooms.get('tvbooksfilms'));
+const yt = /** @type {ChatRoom} */ (Rooms.get('youtube'));
```

Comment:
So what does this do on side servers where these rooms don't exist?

Reply:
The same sort of structure is used in all the other room-specific chat plugins on PS. If those rooms don't exist the plugin should be deleted.

Reply:
I assume the current behavior is just "crash if one of the plugin's commands are used"